code/src/pages/dashboard/dashboard-orders/DashboardOrdersPage.tsx

```diff
@@ -21,14 +23,39 @@ const DashboardOrdersPage = () => {
   const {
     filters,
     filterActions,
+    searchActions,
+    searchFilters,
     activeFiltersCount,
     orders,
     page,
     totalPages,
     isLoading
   } = useFilteredAdminOrders();
 
+  const { applyFilters, resetFilterByKey, updateFilterByKey } = searchActions;
+
   const [isFilterDrawerOpen, setIsFilterDrawerOpen] = useState(false);
+  const [searchValue, setSearchValue] = useState(
+    searchFilters.accountEmail || ""
+  );
```

Comment:
We don't need this state. Use `searchFilters.accountEmail` as value and `updateFilterByKey` as updator function

Reply:
Fixed! Thank you for this advices!